.rubocop.yml

```diff
@@ -257,62 +240,6 @@ Metrics/PerceivedComplexity:
                  human reader.
   Enabled: false
 
-##################### Performance #############################
```

Comment:
The Performance rules were extracted into a separate gem, and the logic around Ruby version differences makes sense, so this just removes the rules.